src/Constants/application.type.ts

```diff
@@ -2,3 +2,46 @@ export interface SinleMenuListHOC {
     IconComponentName: any;
     menuTitle: string;
 }
+
+export interface UIConstantsType {
+    projectHeaderTitle: string;
+    addToCart: string;
+    projectDetails: string;
+    projectDetailsSubmit: string;
+    projectName: string;
+    projectCode: string;
+    projectChannelOwner: string;
+    projectActivitiesLastWeek: string;
+    projectActivitiesNextWeek: string;
+    projectAccomplishment: string;
+    projectStatus: string;
+    projectCompletionDate: string;
+    projectBandwidthAvailble: string;
+    leftMenuHome: string;
+    leftMenuILC: string;
+    leftMenuForum: string;
+    leftMenuProjectDetails: string;
+    leftMenuAdmin: string;
+    leftMenuSettings: string;
+    isRequired: string;
+    maxMessage: string;
+}
```

Comment:
Is this required to specify the types for the UI Constants?

Reply:
It's always better to have strong type safety, pls refer **src/Constants/UIConstants.tsx** or **src/utils/validation-helper.ts**, however, if you recommend, we can remove it. Pls suggest.

Reply:
Strong type safety is required. However, in general, we are storing strings in the Constants file and also we are not trying to alter the Constants value after importing it into other files. Hence it's not absolutely required to specify types for Constants.

Reply:
okay

Reply:
done